README.md

```diff
@@ -61,7 +61,7 @@ To launch the server for TCP sockets use:
 yarn start
 ```
 
-This starts a server that is listening on port 5007 for incoming client requests.
+This starts a server that is listening on port 0 for incoming client requests.
```

Comment:
Should still start  at port 5007. The server cannot be tested standalone, so it has always be tested with an integration project in "external server mode". For this we need an explicit port to connect to.